tests/test_ic_od_to_kvp_wrapper/test_classification_as_kvp.py

```diff
@@ -0,0 +1,71 @@
+import copy
+import json
+import pathlib
+import tempfile
+import unittest
+
+from PIL import Image
+
+from vision_datasets.common import (
+    CocoManifestAdaptorFactory,
+    DatasetInfo,
+    DatasetTypes,
+    VisionDataset,
+)
+from vision_datasets.image_classification.classification_as_kvp_dataset import (
+    ClassificationAsKeyValuePairDataset,
+)
+
+
+class TestMultilcassClassificationDataset:
+    DATASET_INFO_DICT = {
```

Comment:
I see there is already a `DetectionTestFixtures` in `tests.test_fixtures`. Could we move the same elements of this class to a `MulticlassClassificationTextFixtures` (or `ClassificationTextFixtures`, whichever scope makes sense) to that file to keep it organized?

Reply:
Moved.